src/amo/utils/index.js

```diff
@@ -375,13 +375,7 @@ export const normalizeFileNameId = (filename: string): string => {
 };
 
 export const getDisplayName = (component: React.ComponentType<any>): string => {
-  const defaultName = 'Component';
-
-  if (component) {
-    return component.displayName || component.name || defaultName;
-  }
-
-  return defaultName;
+  return component.displayName || component.name || 'Component';
```

Comment:
This file is covered by Flow, and Flow dictates that `component` not be `null` or `undefined`, so I don't believe any of this extra code is needed. I also checked that every file that calls this is also covered by Flow.